packages/util/src/equal.ts

```diff
@@ -1,7 +1,7 @@
 import { isNullish } from './nullish';
 // eslint-disable-next-line valid-jsdoc
 /**
- * hidden
+ * Hidden.
```

Comment:
```suggestion
 * @hidden.
```
See doc: https://typedoc.org/guides/doccomments/#%40hidden-and-%40ignore
Please also adjust other occurrences.

Reply:
We export everything from util package, so I am not sure if we need `@hidden` at all, atleast here. I will remove this and adjust in other places

Reply:
This hidden is for hiding the api for doc generation.
We can also remove the "hidden" and add valid docs.